pylab/website/views.py

```diff
@@ -71,7 +71,26 @@ def about(request):
 
 def event_details(request, year, month, day, slug):
     event = get_object_or_404(Event, starts__year=year, starts__month=month, starts__day=day, slug=slug)
-    return render(request, 'website/event_details.html', {'event': event})
+    attendance = Attendance.objects.filter(event=event)
+    if request.method == 'POST':
+        form = website_forms.AttendanceForm(request.POST)
+        if form.is_valid():
+            instance = Attendance.objects.get(event=event, attendee=request.user)
+            instance.response = form.cleaned_data['response']
+            instance.save()
+            return redirect(event)
+    else:
+        if request.user.is_authenticated():
+            instance, created = Attendance.objects.get_or_create(event=event, attendee=request.user,  # pylint: disable=unused-variable
+                                                                 defaults={'response': 1})
```

Comment:
You don't need ```defaults={'response': 1}``` here, because user didn't select his response yet, but ```response``` value 1 is written to the database. The value should be ```null``` as it would be by default.